src/Pdp/LabelsTrait.php

```diff
@@ -1,34 +1,61 @@
 <?php
-
-declare(strict_types=1);
```

Comment:
Shouldn't all these files include this `declare` statement?

Reply:
yep typo from my part it will be change too

Reply:
declare is present just moved after the credits